archunit-junit/junit5/engine/build.gradle

```diff
@@ -46,23 +51,19 @@ test {
 
 shadowJar {
     exclude 'META-INF/maven/**'
-
-    dependencies {
```

Comment:
Certain dependencies (e.g. `junitPlatformLaungher`) are needed with their original (unshadowed) package name (see why [here](https://github.com/crizzis/ArchUnit/pull/1/files#diff-f69c83fcf7315e10de052719edb3d27c93666f3e4310f2d3235e999b5e69d913))

The changes made to the `shadowJar` config, as well as disabling both `checkExportedPackage` and `checkArtifact`, was the only way I managed to get it work. 

Please advise on how this requirement should be handled. If the packages absolutely must be shadowed, I can introduce a proxy at runtime, but that just adds unnecessary complexity IMHO